plugins/notifications/src/components/NotificationsFilters/NotificationsFilters.tsx

```diff
@@ -47,58 +49,53 @@ export type NotificationsFiltersProps = {
 
 const ALL = '___all___';
 
-export const CreatedAfterOptions: {
-  [key: string]: { label: string; getDate: () => Date };
-} = {
+type TranslationKey = keyof (typeof notificationsTranslationRef)['T'];
+
+export const CreatedAfterOptions = {
   last24h: {
-    label: 'Last 24h',
+    labelKey: 'filters.createdAfter.last24h' as const satisfies TranslationKey,
     getDate: () => new Date(Date.now() - 24 * 3600 * 1000),
   },
   lastWeek: {
-    label: 'Last week',
+    labelKey: 'filters.createdAfter.lastWeek' as const satisfies TranslationKey,
```

Comment:
```suggestion
    labelKey: 'filters.createdAfter.lastWeek' satisfies TranslationKey,
```
@ciiay  is `as const`  is required here ?